storage/file.go

```diff
@@ -34,6 +35,8 @@ type FileProperties struct {
 	Length       uint64 `xml:"Content-Length"`
 	MD5          string `header:"x-ms-content-md5"`
 	Type         string `header:"x-ms-content-type"`
+	CopyID       string `header:"x-ms-copy-id"`
```

Comment:
Instead of adding these fields to the FileProperties struct we should use FileCopyState instead, adding it as a field to the File struct.